.circleci/config.yml

```diff
@@ -415,7 +415,7 @@ jobs:
       - run:
           name: Install APKs
           command: |
-            docker run -d --name fenix-files ${DOCKERHUB_REPO}:fenix-apk-store
+            docker run -d --name fenix-files mozilla/experimenter:fenix-apk-store
```

Comment:
Oh it doens't look like we even need to `run` here since I think that's basically just triggering a pull and then exiting, you could just write `docker pull` here, I think.

Reply:
Will the `cp` do the run? I think I need to run it before I copy from it.

Reply:
`cp` might only work if the image is already available locally, so you'll prolly need to do SOMETHING to get the image locally available, whether it's run with implicit fetch or explicit fetch.

Reply:
What would you like here, I am not help to either solution

Reply:
Just leave it alone for now, if it works it's fine, we can change it later if we need to 👍